Source/WebCore/platform/graphics/cocoa/VideoMediaSampleRenderer.mm

```diff
@@ -520,9 +523,31 @@ static bool isRendererThreadSafe(WebSampleBufferVideoRendering *renderering)
 
     PAL::CMTimebaseSetTimerDispatchSourceNextFireTime(timebase.get(), timerSource.get(), nextPurgeTime, 0);
 
+    m_nextScheduledPurge = nextPurgeTime;
+
     return samplesPurged;
 }
 
+void VideoMediaSampleRenderer::maybeReschedulePurge()
```

Comment:
This is more a `maybePurge` than anything else.

Reply:
yes, but in PR #37128 it really is just a new scheduling. So I will keep this name.